README.md

```diff
@@ -17,7 +17,7 @@ Alternatively,
     
     $ git clone https://github.com/mop-tracker/mop
     $ cd mop
-    $ go build ./mop/cmd
+    $ go build ./cmd/mop
     $ ./mop
```

Comment:
It is my understanding that this would need to be changed as well, correct?

Reply:
If you follow that sequence of commands exactly, then the "./mop" works.

Reply:
Noted, then I shall accept the patch.
Thank you very much.